docs/UserGuide.md

```diff
@@ -108,6 +108,7 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 * When editing tags, the existing tags of the client will be removed i.e adding of tags is not cumulative.
 * You can remove all the client’s tags by typing `t/` without
     specifying any tags after it.
+* Insurance plans and claims cannot be edited.
```

Comment:
Maybe we can state that "for more features related to modifying insurance plans and claims, please refer below". Since we do allow users to "edit" Insurance and Claims but through other features (such as `closeClaim`)

Reply:
Okies, will make the necessary edits!